src/lib/components/Segment.svelte

```diff
@@ -105,12 +105,9 @@
     border-radius: var(--border-radius);
     overflow: hidden;
 
-    width: 100%;
-
-    @include media.min-width(medium) {
-      width: var(--segment-min-width, var(--segment-width, fit-content));
-      --segment-min-width: calc(var(--segment-button-width) * var(--segments));
-    }
+    box-sizing: border-box;
+    width: var(--segment-min-width, var(--segment-width, fit-content));
+    --segment-min-width: calc(var(--segment-button-width) * var(--segments));
```

Comment:
Does it work defining it after being used?

Reply:
Seems so. GPT: In CSS, the order in which you define CSS variables (also known as custom properties) relative to their usage does not generally affect their functionality, thanks to CSS's cascading and scope rules. CSS variables are accessible within the scope they are declared, and they can be used before they are defined in the document flow. This is because CSS parsing does not strictly depend on the order of declaration as it does in some programming languages like JavaScript.